browser_use/browser/profile.py

```diff
@@ -662,6 +662,17 @@ class BrowserProfile(BrowserConnectArgs, BrowserLaunchPersistentContextArgs, Bro
 		default=None, description='File to save cookies to. DEPRECATED, use `storage_state` instead.'
 	)
 
+	# --- Recording Options ---
```

Comment:
```suggestion
    # these shadow the old playwright args on BrowserContextArgs, but it's ok
    # because we handle them ourselves in a watchdog and we no longer use playwright, so they should live in the scope for our own config in BrowserProfile long-term
```